src/calculateRentalCost.js

```diff
@@ -4,7 +4,7 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  return days >= 7 ? days * 40 - 50 : days >= 3 ? days * 40 - 20 : days * 40;
```

Comment:
2. [checklist] Don't use literals in expressions (HARDCODE). Instead use constants explaining this values

Save all values in constants, avoid [magic numbers](https://medium.com/@samaddico/programming-and-magic-numbers-f766e0cd1369)